package.json

```diff
@@ -10,6 +10,10 @@
     "lint": "eslint ."
   },
   "dependencies": {
+    "eslint-config-prettier": "^6.8.0",
+    "eslint-plugin-prettier": "^3.1.2",
+    "lint-staged": ">=8",
+    "prettier": "^1.19.1",
```

Comment:
@anjumpaul these dependencies can be moved to devDependencies section.

Also, turns out that @react-native-community/eslint-config already ships with Prettier and associated config. So we need not have added those. You can uninstall Prettier and associated dependencies. So the only additional package will be lint-staged and that can be moved to devDependencies.

Reply:
Removed Requested changes